### PR TITLE
fix(api): missing base token data in info response

### LINKED ISSUE #803: Missing `BaseToken` from `core/v2/info/`
The `baseToken` field is missing from Chronicle's response.

### PR DESCRIPTION
**Pull request by Alex6323**
## Linked Issues

* Closes #803

## Notes to Reviewer

As a reviewer, please pay particular attention to the following areas when reviewing this PR and tick the above boxes after you have completed the steps.

#### API Changes
* [ ] Test the API endpoints which were added/changed.
* [ ] Ensure that the API response times scale with database size appropriately.
* [ ] Review the API documentation changes and confirm that it matches the actual functionality.
* [ ] Check for breaking changes in the API and matching (conventional) commit message prefix.

#### Test cases
* [ ] Review and run tests that were added/changed.
* [ ] Suggest places that may benefit from test cases.

#### INX Changes
* [ ] Run chronicle using an INX connection.

#### Database Changes
* [ ] Review database queries for correctness/conciseness.
* [x] Ensure queries are supported by indexes if needed.
* [x] Check for breaking changes in the data model and matching (conventional) commit message prefix.

### CODE REVIEW

**Review by grtlr — DISMISSED** [review]
Can you finish the documentation please, otherwise it looks good to me 🙏.

File: src/bin/inx-chronicle/api/stardust/core/routes.rs [hunk 1]
```diff
@@ -169,11 +185,14 @@ pub async fn info(database: Extension<MongoDb>) -> ApiResult<InfoResponse> {
             },
             token_supply: protocol.token_supply.to_string(),
         },
-        status: StatusResponse {
-            is_healthy,
-            latest_milestone,
-            confirmed_milestone,
-            pruning_index: oldest_milestone.milestone_index.0 - 1,
+        // FIXME/TODO
```
[the remark this answers is not in the document]
Comment (Alex6323) [reply src/bin/inx-chronicle/api/stardust/core/routes.rs:188]: Forgot to remove it.

File: src/bin/inx-chronicle/stardust_inx/mod.rs [hunk 2]
```diff
@@ -134,7 +134,19 @@ impl InxWorker {
...
+        let node_configuration = inx.read_node_configuration().await?;
+
+        debug!(
+            "Connected to network `{}` with base token `{}[{}]`.",
+            protocol_parameters.network_name(),
+            node_configuration.base_token.name,
+            node_configuration.base_token.ticker_symbol
+        );
+
+        self.db
```
Comment (Alexandcoats) [inline src/bin/inx-chronicle/stardust_inx/mod.rs:146]: Shouldn't this also be checked per-milestone?

File: src/db/collections/node_configuration.rs [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        MongoDb,
+    },
+    types::node::NodeConfiguration,
+};
+
+/// The corresponding MongoDb document representation to store [`NodeConfiguration`]s.
+#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
+pub struct NodeConfigurationDocument {
+    #[serde(rename = "_id")]
+    id: (),
```
Comment (Alexandcoats) [inline src/db/collections/node_configuration.rs:19]: No reason for this field?

File: src/db/collections/node_configuration.rs [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+        MongoDb,
+    },
+    types::node::NodeConfiguration,
+};
+
+/// The corresponding MongoDb document representation to store [`NodeConfiguration`]s.
+#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
+pub struct NodeConfigurationDocument {
+    pub config: NodeConfiguration,
+}
```
Comment (grtlr) [inline src/db/collections/node_configuration.rs:19]: Why not use a tuple struct as wrapper? We shouldn't have the indirection through `config` in the database. Alternatively we could use `#[serde(flatten)]`.

Comment (Alex6323) [reply src/db/collections/node_configuration.rs:19]: I look into it.

Comment (Alex6323) [reply src/db/collections/node_configuration.rs]: Decided to go with the new-type solution.

**Review by Alexandcoats — APPROVED** [review]

**Pull request merged**